spec/models/Restaurant_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'rails_helper'
+require 'faker'
+
+describe Restaurant do
+
+  it { should validate_presence_of(:name) }
+  it { should have_many(:reviews) }
+  it { should belong_to(:cuisine) }
+
+  it { should allow_values('SomeString').for(:address) }
+  it { should_not allow_values(Faker::Lovecraft.fhtagn(5)).for(:address) }
+
+  it { should allow_values('35').for(:delivery_time) }
```

Comment:
Delivery time should be an integer in the range of 1 to 1440.'35' and '5000' doesn't meet this criteria, the test should have failed.

Reply:
Maybe if you defined the delivery time as numericality: { only_integer: true } in the model the test would have failed, you can try that